requirements_dev.txt

```diff
@@ -2,4 +2,6 @@
 black==23.3.0
 mypy==1.3.0
 pytest-cov==4.1.0
-coverage==7.2.3
\ No newline at end of file
+coverage==7.2.7
+types-jsonschema>=4.17
+types-requests>=2.31
```

Comment:
lets make these into exact versions just in case, so we can eliminate one potential place for a bug

Reply:
updated this!